### ISSUE TITLE
VirtualRendering:false is not working

### ISSUE THREAD

**Issue by monarchjhaveri** [zippyui/react-datagrid#102]
The `virtualRendering={false}` prop does not have any effect; as far as I can tell, the grid still uses virtualRendering. Could you please fix?

**Comment by radubrehar** [comment]
Yes, I'm going to look into this next week. Try `rowHeight=null` until then and see if it works

**Comment by monarchjhaveri** [comment]
Looks like the `rowHeight=null` workaround works
however, it seems performance is pretty bad (as would be expected :-)) on a table with 18 (plain text) columns and 500 rows. Would you say this is normal? Or maybe its on my end?

**Comment by monarchjhaveri** [comment]
update: without any apparent reason, performance on `virtualRendering=false` datagrid just dropped and is unusable (18 cols 500 rows)

**Comment by radubrehar** [comment]
Please make sure you try with the production version of React, since the dev version has a lot of checks, which can really slow down the app in development.

**Comment by monarchjhaveri** [comment]
You are right, turns out we were in dev mode and not prod mode. Would still like to see virtualRendering flag working :-)

**Comment by radubrehar** [comment]
Glad to hear you got it working. Will have a look at `virtualRendering` prop

**Comment by nikhilk78** [comment]
There is a function isVirtualRendering which checks for props.virtualRendering || props.rowHeight != null
if we fix this to props.virtualRendering || props.rowHeight != null && props.virtualRendering  != false should enable virtualRendering and still enable to specify row height, I think.
By the way, I am new to whole webapps, how do I use React in PRODUCTION environment as mentioned in earlier post?
Thanks

**Comment by nikhilk78** [comment]
I think I found it npm config set NODE_ENV Production, am I right?

**Comment by monarchjhaveri** [comment]
@nikhilk78 yes

**Comment by pruik** [comment]
+1
I would like to disabled virtualRendering while still having a fixed height of 50px for each row
